### PR TITLE
New events

### PR DESCRIPTION
**Pull request by tonywu94**
# :ship: Pull Request

## :question: Purpose

New events (minus the win lose events)

**Give a brief description of your changes:**

## :hammer: Validation Strategy

**Describe how you verified your changes (if applicable):**

## :tickets: Ticket(s)

Affects #

(Create a copy of that line for each Github Issue affected,
and replace "Affects" with "Closes" if merging this will close the relevant ticket.)

### CODE REVIEW

File: src/choices.json [hunk 1]
```diff
@@ -126,34 +126,31 @@
...
+  "Kinesiology": {
+    "answer": "Kinesiology",
+    "followUp": "",
+    "statChanges": [100, 2.0, 100],
     "minigame": "",
     "dlogo": ["KIN", "#228B22", "#FFFFFF"],
     "btnStyle": "kin"
   },
-  
+
```
Comment (eloisepeng) [inline src/choices.json]: can you remove this line for me please. sorry I overlooked

File: src/choices.json [hunk 2]
```diff
@@ -198,5 +195,75 @@
...
+  },
+  "ScaleWall": {
+    "answer": "I can totally scale 10.5 ft wall by myself",
+    "followUp": "",
+    "statChanges": [20, 0, 0],
+    "minigame": "",
+    "btnStyle": ""
+  },
+  "CantReachTopShelf": {
+    "answer": "I can't even reach the top shelf... 🥺",
```
Comment (eloisepeng) [inline src/choices.json]: wow, we can do emoji?

**Comment by cheukyin699** [comment]
image

bug???

File: src/events.json [hunk 3]
```diff
@@ -65,10 +65,52 @@
 	},
 	"FratPartyEvent": {
 		"prompt": "You are invited to a toga party! Are you going to go?",
+		"imgPath": "FratPartyEvent/toga.png",
+		"choices": [
+			"UseWheels",
+			"SquirrelPictures"
```
Comment (cheukyin699) [inline src/events.json]: Copy and paste issue? Probably not supposed to do that. May want to check to see if any other choices are mis-labelled or mis-assigned.

**Comment by eloisepeng** [comment]
> image
> 
> bug???

progress bar you mean?

**Comment by cheukyin699** [comment]
"You are invited to a toga party"

Choices:
- Puts on helmet???
- I gotta get pictures???

**Comment by eloisepeng** [comment]
> "You are invited to a toga party"
> 
> Choices:
> 
> * Puts on helmet???
> * I gotta get pictures???

lol, the json must got messed up

File: src/events.json (hunk 3 above)
Comment (eloisepeng) [reply src/events.json]: good catch

Comment (tonywu94) [reply src/events.json]: I think this was a merge issue. Stuff got really weird with all the changes to the JSON in other branches

File: src/events.json [hunk 4]
```diff
@@ -74,23 +74,85 @@
...
+		"hasInnerFill": true,
+		"gamePlayMode": "Comics"
+	},
 	"SquirrelEvent": {
 		"prompt": "You have a lecture across campus but there are squirrels in the way!",
 		"imgPath": "",
 		"choices": [
-			"UseWheels",
-			"SquirrelPictures"
+			"UseWheels"
```
Comment (cheukyin699) [inline src/events.json]: There should be another choice here.

File: src/events.json (hunk 3 above)
Comment (eloisepeng) [reply src/events.json]: which one was it? I compared every line in my last merge. maybe some commit before that?

File: src/events.json (hunk 4 above)
Comment (tonywu94) [reply src/events.json]: Let's leave the choice out so that the player has to play the minigame?

Comment (tonywu94) [reply src/events.json]: aka don't give them an option to skip?

File: src/events.json (hunk 3 above)
Comment (tonywu94) [reply src/events.json]: eh I dunno, things went weird

File: src/events.json (hunk 4 above)
Comment (cheukyin699) [reply src/events.json]: sure that would work.

**Pull request merged**